enhancements/console/custom-logos.md

```diff
@@ -0,0 +1,151 @@
+
+This feature should allow users to add custom logos for the masthead and favicon that are compatible with both light and dark theme modes in the OpenShift web console.
+
+### Non-Goals
+
+
+## Proposal
+
+### Workflow Description
+
+├── spec
+│   ├── customization
+│       ├── customLogos
+│           ├── type
+│           ├── themes
+│               ├── type
+│               ├── file
+│                   ├── key
+│                   ├── name
+└── ...
+
+### Risks and Mitigations
+
+1. Users could set both the `CustomLogoFile` and `CustomLogos` APIs. When both APIs are set. 
+
+To mitigate this challenge, the `CustomLogos` API configuration will take precedence over the old `CustomLogoFile` field.
+
+2. Handling different custom logos for light and dark themes may increase the complexity of the backend and UI logic. 
+
+To mitigate this challenge, thoroughly test and validate the new logic, and implement fallback mechanisms to ensure logos are always visible if any are missing.
+
+3. Users might experience confusion with the introduction of new logo configuration options, especially if they are familiar with the old method, which may soon be deprecated.  
+
+To mitigate this challenge, provide comprehensive documentation and tools that guide users through the transition. Include clear instructions about the changes and their benefits.
+
+
+### Drawbacks
+
+N/A
+
+## API Design Details
+
+The configuration for custom logos will include support for both masthead and favicon types, with separate files for light and dark themes:
+
+```yaml
+customLogos:
+  - type: Masthead
+    themes:
+      - type: Light
+        file:
+          key: logo-light.svg
+          name: masthead-logo-light
+      - type: Dark
+        file:
+          key: logo-dark.svg
+          name: masthead-logo-dark
+  - type: Favicon
+    themes:
+      - type: Light
+        file:
+          key: favicon-light.ico
+          name: favicon-logo-light
+      - type: Dark
+        file:
+          key: favicon-dark.ico
+          name: favicon-logo-dark
+```
+
+### Attributes Description
+
+#### customLogos
+- `type`: Enumerate which specifies the type of custom logo. Available custom logo types are `Masthead` and `Favicon`.
+- `themes`: A list of themes for which the custom logo is defined.
+
+#### themes
+- `type`: Enumerate which specifies the type of theme. Available theme types are `Light` and `Dark`.
+- `file`: Contains the file details for the custom logo.
+
+#### file
+- `key`: The key or path to the logo file.
+- `name`: The name identifier for the custom logo.
+
+### Test Plan
+
+### Graduation Criteria
+
+N/A
+
+
+#### Dev Preview -> Tech Preview
+
+N/A
+
+#### Tech Preview -> GA
+
+N/A
+
+#### Removing a deprecated feature
+
+The current custom logo feature is deprecated and will be removed at some point in the future. Users are encouraged to transition to the new custom logo configuration that supports light and dark theme modes for the masthead and favicon.
```

Comment:
It's not clear to me what API tier our operator config is, but I wouldn't expect us to remove the deprecated field anytime soon. It's not a huge burden to support both. If it's tier 1, we aren't able to remove it until the next major version.

https://docs.openshift.com/container-platform/4.17/rest_api/overview/understanding-api-support-tiers.html